.github/workflows/test.yml

```
@@ -8,35 +8,35 @@ jobs:
 
-    - name: Send coverage
-      uses: shogo82148/actions-goveralls@25f5320d970fb565100cf1993ada29be1bb196a1 # v1
-      with:
-        path-to-profile: coverage.out
-        flag-name: Go-${{ matrix.go }}
-        parallel: true
+      - name: Send coverage
+        uses: shogo82148/actions-goveralls@25f5320d970fb565100cf1993ada29be1bb196a1 # v1
+        with:
+          path-to-profile: coverage.out
+          flag-name: Go-${{ matrix.go }}
```

actionlint (1.7.12)

[error] property "go" is not defined in object type {go-version: string; os: string}

(expression)